requirements.txt

```diff
@@ -0,0 +1,3 @@
+argcomplete>=3.5.2
```

Comment:
This one is of debatable merit, but hopefully benign enough to justify its inclusion.